src/extension.ts

```diff
@@ -60,6 +60,9 @@ export function activate(context: ExtensionContext): void {
 
   // start the client and the server
   client.start();
+  client.onNotification('ansible/showMessage', async (message) => {
```

Comment:
The notification message send from the server is `ansible/errorMessage`  whereas at this line the message checked in `ansible/showMessage`. Is this expected? Also if the server sends `ansible/showMessage` notification it will be displayed as an error message in the window which doesn't seem right. The logic can be updated to reflect below mapping?
 
`ansible/errorMessage` -> `window.showErrorMessage`
`ansible/showMessage` -> `window.showInformationMessage`